MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -46,23 +50,35 @@
 
 mfvVertices = cms.EDProducer('MFVVertexer',
                              kvr_params = kvr_params,
+                             do_track_refinement = cms.bool(False), # remove tracks + trim out tracks with IP significance larger than trackrefine_sigmacut and trackrefine_trimmax, respectively   
+                             resolve_split_vertices_loose = cms.bool(False), # an alternative merging routine with loose criteria 
+                             resolve_split_vertices_tight = cms.bool(True), # new merging routine 
+                             investigate_merged_vertices = cms.bool(True), # investigate quality cuts on new merged vertices from tight merging 
```

Comment:
Do we want this on always?

Reply:
We need `resolve_split_vertices_tight = cms.bool(True)` but `investigate_merged_vertices = cms.bool(True)` can be off.

Reply:
Done.